<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperatorClass</class>
 <widget class="QMainWindow" name="OperatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Obs Operator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="sceneSelect">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string extracomment="场景选择">s1</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>s1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>s2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>s3</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>setting</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tcp Server</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="svrTcpHost">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.11.52</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Host</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="svrTcpPort">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>13010</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>devId</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="devId">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>obs</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>streamId</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="streamId">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>pk01</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>scene select</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>751</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>60</y>
      <width>16</width>
      <height>571</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Operator.qrc"/>
 </resources>
 <connections/>
</ui>
